File: clients/csharp/src/Swagger/Client/Model/leaderboard.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Swagger.Client.Model {
  public class Leaderboard {
    public string username { get; set; }

    public double? marginBalance { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Leaderboard {\n");
      sb.Append("  username: ").Append(username).Append("\n");
      sb.Append("  marginBalance: ").Append(marginBalance).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: java/src/main/java/com/wordnik/client/model/leaderboard.java
package com.wordnik.client.model;

public class Leaderboard {
  private String username = null;
  private Double marginBalance = null;
  public String getUsername() {
    return username;
  }
  public void setUsername(String username) {
    this.username = username;
  }

  public Double getMarginBalance() {
    return marginBalance;
  }
  public void setMarginBalance(Double marginBalance) {
    this.marginBalance = marginBalance;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Leaderboard {\n");
    sb.append("  username: ").append(username).append("\n");
    sb.append("  marginBalance: ").append(marginBalance).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

